docs/Committers.md

```diff
@@ -85,6 +85,7 @@ instead, it should be brought up with the Community Governance Committee.
 | Anand Krishnamoorthi  | anakrish            | Debugging, SGX, EDL, Dev Tools |
 | Andrew Schwartzmeyer  | andschwa            | EDL, CMake, Git, Dev Tools     |
 | Brett McLaren         | BRMcLaren           | Build, CMake, CI               |
+| Vikas Tikoo           | vtikoo              |                                |
```

Comment:
I don't think any committer should have 0 areas of responsibility, so that needs to be filled in.

Also, I think going forward that for someone to get committer permissions, they should have a history of quality PR submissions and/or reviews and/or issues filed.   Such justification should be added to the PR description.